test/powershell/Modules/Microsoft.PowerShell.Management/Start-Process.Tests.ps1

```diff
@@ -69,5 +68,17 @@ Describe "Start-Process" -Tags @("CI","SLOW") {
 	$dirEntry.Length | Should BeGreaterThan 0
     }
 
+    It "Should give an error when Verb parameter is used" -Skip:(-not $IsCoreClr) {
+        try 
+        {
+            Start-Process -Verb runas -FilePath $pingCommand -ArgumentList $pingParam -ErrorAction Stop
```

Comment:
`-Verb` would trigger a terminating error, so `-ErrorAction stop` is unnecessary here.